pkg/asset/manifests/openshift.go

```diff
@@ -307,6 +309,7 @@ func (o *Openshift) Generate(dependencies asset.Parents) error {
 	}
 
 	o.FileList = append(o.FileList, openshiftInstall.Files()...)
+	o.FileList = append(o.FileList, featureGate.Files()...)
```

Comment:
Since FeatureGate's Generate() could return nil, do we want to check for that before calling the append()?

Reply:
`append` should gracefully handle this case:
https://go.dev/play/p/tOrMzBn_wS_I